app/views/courtSearchInfo.html

```diff
@@ -1,34 +1,37 @@
-<div class="container" id="courtSearchInfoPage">
-  <div class="row">
-    <div class="col-sm-12">
-      <h2>{{ctrl.courtName}} Court Information</h2>
-        <div class="row" style="margin-top: 20px;">
-          <div class="col-sm-6">
-            <h3>Court Address</h3>
-            <p class="address1">{{ctrl.courtInfo.address}}</p>
-            <p class="address2">{{ctrl.courtInfo.city}}, {{ctrl.courtInfo.state}} {{ctrl.courtInfo.zip_code}}</p>
-
-            <div ng-if="ctrl.courtPhone != '' ">
-              <h3>Court Phone Number</h3>
-              <p>{{ctrl.courtPhone}} <span ng-if="ctrl.phoneExtension != ''"> x{{ctrl.phoneExtension}}</span></p> <!-- Maybe we need to format the numbers -->
-            </div>
+<div class="sub-page-title-block">
+  <h1>{{ctrl.courtName}} Court Information</h1>
+</div>
+<div class="sub-page-container" id="courtSearchInfoPage">
+  <div class="container-fluid topMargin alignContainerToPage">
+   <div class="row alignRowToPage">
+      <div class="col-sm-6 alignColumnsToPageToPage">
+        <h2 class="no-indent">Court Address</h2>
+        <p class="address no-indent">{{ctrl.courtInfo.address}}</p>
+        <p class="address no-indent">{{ctrl.courtInfo.city}}, {{ctrl.courtInfo.state}} {{ctrl.courtInfo.zip_code}}</p>
 
-            <div ng-if="ctrl.courtWebsite != '' ">
-              <h3>Website</h3>
-              <p ng-bind-html="ctrl.courtWebsite | linky"></p>
-            </div>
+        <div class="topMargin" ng-if="ctrl.courtPhone != '' ">
```

Comment:
In JavaScript, empty string is falsy. you can change these ng-if's to just "!"....e.g. `ng-if="ctrl.courtPhone"`....for all of them